### PR TITLE
Fixes - issue #3437

### PR DESCRIPTION
**Pull request by tsl143**
Fixes - issue #3437

### CODE REVIEW

File: static/css/impala/addon_details.less [hunk 1]
```diff
@@ -147,6 +147,15 @@ h1.addon,
     &.dependencies {
         background-color: #f8ffdc;
         border-color: #dbe2bf;
+
+        .extra{
```
Comment (tofumatt) [inline static/css/impala/addon_details.less]: Please add spaces before all `{`

File: static/css/impala/addon_details.less [hunk 2]
```diff
@@ -147,6 +147,15 @@ h1.addon,
...
+
+            .more-versions{
+                a{
+                    color: #0996F8;
```
Comment (tofumatt) [inline static/css/impala/addon_details.less]: Can you make the letters lowercase to match the rest of the file? Thanks 😄

**Comment by tofumatt** [comment]
Could you please attach before/after screenshots so QA can test this?

**Comment by tsl143** [comment]
Before
screenshot from 2016-09-11 19 14 05

After
screenshot from 2016-09-11 19 15 00

File: static/css/impala/addon_details.less [hunk 3]
```diff
@@ -147,6 +147,15 @@ h1.addon,
     &.dependencies {
         background-color: #f8ffdc;
         border-color: #dbe2bf;
+
+        .extra {
+
+            .more-versions {
```
Comment (tofumatt) [inline static/css/impala/addon_details.less]: This doesn't need to be a sub-selector; `.extra .more-versions` would work.

**Comment by tofumatt** [comment]
If we could drop that link to a new line it might be nice. If we made it `clear: both;` would that do the trick? Centring the text would be nice too; I know it's a bit out of scope of the bug but I think it'd make it a lot more visually pleasing 😄

**Comment by tsl143** [comment]
Giving inline-block to `a` did the trick, and removing padding from inner span to get center alignment.

**Comment by tofumatt** [comment]
Could you upload a new screenshot for QA? Looks good otherwise! r+wc

**Comment by tsl143** [comment]
screenshot from 2016-09-12 17 26 29

**Comment by tsl143** [comment]
@tofumatt , Is there anything else to this, or can we get this merged?

**Comment by tofumatt** [comment]
Oh shoot, sorry, I must have missed this. r+

Sorry, it won't make it onto production until next Thursday now, my bad!

**Comment by atsay** [comment]
Thanks Trishul! And also for adding this to the [recognition](https://wiki.mozilla.org/Add-ons/Contribute/Recognition#September_2016) page yourself :-)

**Pull request merged**